### ISSUE TITLE
Build development (debug) packages and binaries

### ISSUE THREAD

**Issue by AlekSi** [FerretDB/FerretDB#2178]
### What should be done?

After #2170, it should be easy to build and release development (debug) packages and binaries.

### Where?

Taskfile, CI configuration, release checklist, etc.

### Definition of Done

Those packages and binaries are provided.

**Comment by AlekSi** [comment]
Done in another issue / PR
